tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -40,12 +55,14 @@ def test_fields(self):
         self.assertEqual(uut.marker, ("##", "#", "#"))
         self.assertEqual(uut.range, None)
 
+    def test_not_implemented(self):
+        not_implemented = DocumentationComment(
+            "some docs", "nolang", "doxygen", None, None, None)
+        with self.assertRaises(NotImplementedError):
+            not_implemented.parse()
 
-class PythonDocumentationCommentTest(unittest.TestCase):
 
-    Description = DocumentationComment.Description
-    Parameter = DocumentationComment.Parameter
-    ReturnValue = DocumentationComment.ReturnValue
+class PythonDocumentationCommentTest(DocumentationCommentTest):
```

Comment:
so well, if you inherit from that class which has test methods as well, those methods will be inherited and pytest will run all of those twice, right? Can you check that by letting one fail?